pkg/operator/nfs/controller.go

```diff
@@ -158,8 +167,8 @@ func (r *NFSServerReconciler) reconcileNFSServerConfig(ctx context.Context, cr *
 		nfsGaneshaConfig := `
 EXPORT {
 	Export_Id = ` + fmt.Sprintf("%v", id) + `;
-	Path = /` + claimName + `;
```

Comment:
how about making export as a constant, since it is getting used at multiple places.